app/serializers/costs_parameters_serializer.rb

```diff
@@ -244,7 +356,7 @@ def costs_storage_and_conversion
   # This group only needs subtotal group queries
```

Comment:
What I don't understand is why still a difference is made between `groups_with_subtotal` and `groups_without_subtotal`. It was my understanding that all subtotals were removed?